### ISSUE TITLE
AMC pruner

### ISSUE THREAD

**Issue by dingguodong-826** [microsoft/nni#4030]
nni is a great work,i pruner InceptionV3 by AMC pruner,but there is a  error 
![image](https://user-images.githubusercontent.com/57715990/128443000-dd448665-a5b3-4a43-86ab-61e02a6b80ee.png)
![image](https://user-images.githubusercontent.com/57715990/128443042-d40ee70b-bdb8-4de9-bdb9-75c71d54b5d7.png)
i try to solve this error by bedug,but i faliure,i use this mythod and faliure too,
https://stackoverflow.com/questions/57187680/how-to-resolve-valueerror-found-array-with-dim-4-estimator-expected-2
i have  no any solution ,hope your reply!

**Comment by linbinskn** [comment]
I haven't reproduced this error. Can you tell me the specific setting of AMC pruner or post some example codes?

**Comment by dingguodong-826** [comment]
Thanks for your reply, I didn't make any special Settings for AMC, I just made some simple changes：
![image](https://user-images.githubusercontent.com/57715990/129824617-f2a111e9-be2e-49cb-ac0d-88b88519c7fc.png)
![image](https://user-images.githubusercontent.com/57715990/129824650-2927b6fc-b3f4-43b1-9ec7-2b1439d1d708.png)
Then I downloaded the inceptionv3 code and the pre-training model from the official website
![image](https://user-images.githubusercontent.com/57715990/129824709-015b901b-ece1-4a7a-ae24-3173193fcd59.png)
Then run amc_search.py

**Comment by dingguodong-826** [comment]
> I haven't reproduced this error. Can you tell me the specific setting of AMC pruner or post some example codes?
Thanks for your reply, I didn't make any special Settings for AMC, I just made some simple changes：
image
image
Then I downloaded the inceptionv3 code and the pre-training model from the official website
image
Then run amc_search.py,It would be appreciated if you could provide a procedure for InceptionV3 pruning

**Comment by dingguodong-826** [comment]
> I haven't reproduced this error. Can you tell me the specific setting of AMC pruner or post some example codes?
Thanks for your reply, I didn't make any special Settings for AMC, I just made some simple changes：
![image](https://user-images.githubusercontent.com/57715990/129825968-1db5f77c-1026-43e5-90a7-cbc3de35cd71.png)
![image](https://user-images.githubusercontent.com/57715990/129825994-8b58864a-c853-43f4-b77d-a6c79a45f8a1.png)
Then I downloaded the inceptionv3 code and the pre-training model from the official website
![image](https://user-images.githubusercontent.com/57715990/129826019-bd7d76b2-09b2-4448-84f4-c40ab9d2e995.png)
Then run amc_search.py，It would be appreciated if you could provide a procedure for InceptionV3 pruning

**Comment by dingguodong-826** [comment]
> I haven't reproduced this error. Can you tell me the specific setting of AMC pruner or post some example codes?

This is the code and pre-training model I used for InceptionV3：
链接：https://pan.baidu.com/s/1yfx7x0O4S-XC61sSyQeEkw 
提取码：1234

**Comment by dingguodong-826** [comment]
> I haven't reproduced this error. Can you tell me the specific setting of AMC pruner or post some example codes?
执行完amc_search.py得到的结果。
链接：https://pan.baidu.com/s/1XychXh0jPMv9ltvcOi7ZGg 
提取码：1234

**Comment by zheng-ningxin** [comment]
This problem is caused by sklearn. sklearn doesn't support regression on the input and output that has more than two dimensions, Currently, we can just skip this function to workaround this problem, as we discussed offline. 
If this solved your problem, please close this issue. @dingguodong-826
